### PR TITLE
Set booking start end time

### PR DESCRIPTION
**Pull request by josh-dadswell-hackney**

### CODE REVIEW

File: RepairsApi.Tests/V2/Services/DRS/DrsMappingTests.cs [hunk 1]
```diff
@@ -422,6 +421,11 @@ private static void VerifyCompleteOrder(updateBooking updateBooking, WorkOrder w
                 .Excluding(o => o.theBusinessData)
             );
             updateBooking.updateBooking1.theBooking.theOrder.status.Should().Be(orderStatus.COMPLETED);
+
+            var expectedStart = workOrder.ClosedDate - (booking.assignedEnd - booking.assignedStart) ?? booking.assignedStart;
+            var expectedEnd = workOrder.ClosedDate ?? booking.assignedEnd;
```
Comment (LBHBomotoso) [inline RepairsApi.Tests/V2/Services/DRS/DrsMappingTests.cs:426]: Isn't this in the usecase?

Comment (josh-dadswell-hackney) [reply RepairsApi.Tests/V2/Services/DRS/DrsMappingTests.cs:426]: No the use case fires a completion event which is picked up by the DRSNotificationHandler, this change is making sure the end time of the booking is whatever was selected by the user when they closed the work order. The start time is then shifted by the duration.

Comment (josh-dadswell-hackney) [reply RepairsApi.Tests/V2/Services/DRS/DrsMappingTests.cs:426]: This is specifically about building the SOAP request for sending to DRS

Comment (LBHBomotoso) [reply RepairsApi.Tests/V2/Services/DRS/DrsMappingTests.cs:426]: cool, understood

**Review by LBHBomotoso — APPROVED** [review]

**Pull request merged**